src/epic/web/index.scss

```diff
@@ -0,0 +1,222 @@
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin-top: $gs-baseline * 2;
+
+    // Horizontal alignment
+    align-items: left;
+
+    @include mq($from: mobileLandscape) {
+        flex-direction: row;
+
+        // Now vertical alignment, because the flex-direction is row
+        align-items: center;
+    }
+}
+
+.contributions__contribute {
+    background-color: $news-main-1;
+    color: #ffffff;
+    font-weight: bold;
+    margin-top: $gs-baseline;
+    text-align: left;
+    padding-left: $gs-gutter * (3 / 4);;
+    padding-right: 0;
+    margin-left: 0;
+    &:hover {
+        background-color: darken($news-main-1, 5%);
+        text-decoration: none;
+    }
+
+    &:after {
+        content: '';
+
+        display: inline-block;
+        background-repeat: no-repeat;
+        width: 30px;
+        height: 20px;
+        background-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40"><path fill="white" stroke="white" d="M22.8 14.6L15.2 7l-.7.7 5.5 6.6H6v1.5h14l-5.5 6.6.7.7 7.6-7.6v-.9"/></svg>');
+        background-size: 100%;
+        float: right;
+    }
+}
+
+a[href].contributions__contribute.contributions__contribute--epic {
+    margin-top: 0;
+    background-color: $news-garnett-highlight;
+    color: $guardian-brand-dark;
+
+    svg path {
+        fill: $guardian-brand-dark; 
+    }
+
+    &:hover {
+        background-color: #b3a000;
+        color: $guardian-brand-dark;
+    }
+
+    &:after {
+        background-image: url('data:image/svg+xml;charset=utf8,%3Csvg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40"%3E%3Cpath fill="$neutral-1" stroke="$neutral-1" d="M22.8 14.6L15.2 7l-.7.7 5.5 6.6H6v1.5h14l-5.5 6.6.7.7 7.6-7.6v-.9"/%3E%3C/svg%3E');
+    }
+}
+
+.contributions__contribute--epic-member {
+    margin: $gs-baseline/3 $gs-gutter/2 $gs-baseline/3 0;
+}
+
+.contributions__payment-logos {
+    height: 25px;
+}
+
+.contributions__contribute--epic-member {
+    margin: $gs-baseline/3 $gs-gutter/2 $gs-baseline/3 0;
+}
+
+.epic__testimonial-container {
+    margin-top: $gs-gutter;
+    margin-bottom: $gs-gutter;
+}
+
+.epic__testimonial-quote {
+    position: relative;
+}
+
+.epic__testimonial-quote span {
+    position: absolute;
+}
+
+.epic__testimonial-container .epic__testimonial-quote .inline-garnett-quote svg {
+    fill: $multimedia-main-2;
+    width: $gs-gutter;
+    height: $gs-gutter;
+    padding-left: 10px;
+}
+
+.epic__testimonial-text {
+    margin-bottom: 0;
+    font-family: $f-sans-serif-text;
+    font-style: italic;
+    padding-left: 2 * $gs-gutter;
+}
+
+.epic__testimonial-test--frontend {
+    line-height: 1.4;
+    margin: 16px 0 28px 24px;
+    @include mq(tablet) {
+        margin-left: 28px;
+    }
+}
+
+.epic__testimonial-name {
+    display: block;
+    margin-top: 5px;
+    font-family: $f-sans-serif-text;
+    font-style: normal;
+}
+
+.epic__testimonial-name--frontend {
+    font-size: .875rem;
```

Comment:
in other codebases we keep stuff in pixels and use a `px-to-rem` plugin - not sure if it's included in this codebase though